fabfile/use.py

```diff
@@ -39,8 +43,12 @@ def enterprise(*args):
     utils.add_github_to_known_hosts() # make sure ssh doesn't prompt
     if len(args) != 1:
         abort('You must provide a single argument, with a command such as "use.enterprise:v6.0.1"')
+    
     git_ref = args[0]
 
+    # register that we use enterprise
+    config.settings['community-enterprise'] = 'enterprise'
```

Comment:
I think it might be easier to set the repo path and build function here so that you dont have to write the if else checks in other methods.

```python
config.settings[CITUS_REPO_PATH] = CITUS_ENTERPRISE_REPO
config.settings[CITUS_BUILD_FUNC] = build_enterprise
```

you can do the same for community version:

```python
config.settings[CITUS_REPO_PATH] = CITUS_REPO
config.settings[CITUS_BUILD_FUNC] = build_citus
```

then in valgrind you can directly use `config.settings[CITUS_BUILD_FUNC]` and `config.settings[CITUS_REPO_PATH]`

does that sound good to you?

Reply:
makes sense. done